website/templates/documentation.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
This could be removed
The api is self-documenting

Reply:
Ok because you did add docs to your landing page that's why i thought i should add it